ja/core-libraries/global-constants-and-functions.rst

```diff
@@ -90,6 +90,86 @@ CakePHP アプリケーション内の特定のファイルやディレクトリ
     $domain でドメインを指定し、 $count の数を数え、 $singular と $plural
     に基いて複数形を正しく処理したメッセージを返します。
 
+.. php:function:: __x(string $context, string $singular, mixed $args = null)
+
+    コンテキストは、同じドメイン内で一意にする翻訳文字列の一意の識別子です。
```

Comment:
The `context` may be better not to be translated, as it would be the argument name.